README.md

```diff
@@ -37,48 +37,48 @@ This project chooses best practices as defaults:
-| Coloured diffing                                                                          | ✔️       | ✔️                                                           | ✖️                                                                |
+| Assertion-specific output with explanation                                                | ✔️       | ✔️                                                           | [Lib](https://hackage.haskell.org/package/tasty-hedgehog)        |
+| Coloured diffing                                                                          | ✔️       | ✔️                                                           | C                                                                |
 | multi-line diffing                                                                        | ✔️       | ✖️                                                           | ✖️                                                                |
 | Contextual failures                                                                       | ✔️       | ✖️                                                           | ✖️                                                                |
-| Inter-test progress output during test suite execution                                    | ✔️       | ✔️                                                           | ?                                                                |
-| Intra-test progress output during test suite execution                                    | 🚧      | ✔️                                                           | ?                                                                |
+| Inter-test progress output during test suite execution                                    | ✔️       | ✔️                                                           | ✔️                                                                |
+| Intra-test progress output during test suite execution                                    | 🚧      | ✔️                                                           | [Lib](https://hackage.haskell.org/package/tasty-quickcheck)      |
 | Optional standard output and standard error suppression [2]                               | ✖️       | ✖️                                                           | ✖️                                                                |
-| Aquire and release a resource for every test in a group (`before` and `after`)            | ✔️       | ✔️                                                           | ✖️                                                                |
+| Aquire and release a resource for every test in a group (`before` and `after`)            | ✔️       | ✔️                                                           | C                                                                |
```

Comment:
I made this C because I can imagine a function that uses `withResource` on every leaf of a `TestTree`

Reply:
A `TestTree` cannot be manipulated, so if you're expecting a `TestTree -> TestTree` transformation, that's not possible. You would have to do this manually yourself (usually by having a `[a -> TestTree]` and then `map`ping them all to use `withResource`)

Reply:
I guess that would count as a `C`, thanks!